src/explore-education-statistics-frontend/src/modules/find-statistics/components/FilterTag.module.scss

```diff
@@ -0,0 +1,24 @@
+@import '~govuk-frontend/govuk/base';
+
+.tag {
```

Comment:
Might want to name this `.button` if we rename the component (see other comment)